.github/workflows/ci-dgraph-upgrade-fixed-versions-tests.yml

```diff
@@ -0,0 +1,40 @@
+name: ci-dgraph-upgrade-fixed-versions-tests
+on:
+  schedule:
+    - cron: "0 */12 * * *" # every 12hrs
+jobs:
+  dgraph-upgrade-fixed-versions-tests:
+    runs-on: [self-hosted, x64]
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Get Go Version
+        run: |
+          #!/bin/bash
+          GOVERSION=$({ [ -f .go-version ] && cat .go-version; })
+          echo "GOVERSION=$GOVERSION" >> $GITHUB_ENV
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ env.GOVERSION }}
+      - name: Make Linux Build and Docker Image
+        run: make docker-image
+      - name: Clean Up Environment
+        run: |
+          #!/bin/bash
+          # clean cache
+          go clean -testcache
+      - name: Run Upgrade and Integration Tests
+        run: |
+          #!/bin/bash
+          # go env settings
+          export GOPATH=~/go
+          # move the binary
+          cp dgraph/dgraph ~/go/bin/dgraph
+          # run the tests
+          go test -v -timeout=4h -failfast -tags=upgrade ./...
```

Comment:
Where do you specify that these are "fixed versions" tests?

Reply:
By default we run the "fixed versions" workflow. If "DGRAPH_UPGRADE_MAIN_ONLY" is set to "true", then we run the workflow for last released version to current main/pr/

Reply:
got it.